src/components/ReportDetailed/index.js

```diff
@@ -19,13 +21,27 @@ type ReportDetailedType = {
 
 type ReportDetailedState = {
   opened: boolean,
+  printable: any, 
 };
 
 class ReportDetailed extends Component<ReportDetailedType, ReportDetailedState> {
+  
+  constructor () {
+    super();
+    this.onChangePrintView = this.onChangePrintView.bind(this);
+  }
+
   state = {
     opened: false,
+    printable: {date: true, start: true, end: true, duration: true, project: true, notes: true }
   };
 
+  onChangePrintView = (data: any, column: any) => {
```

Comment:
I prefer to describe the shape of `data` here. As my other comment mentioned: you can now also get column from here.